src/components/IFrame.js

```diff
@@ -25,7 +28,7 @@ export default class IFrame extends React.Component<Props, State> {
   uuid: string;
   hostName: string;
 
-  constructor(props:Props){
+  constructor(props: Props) {
     super(props);
 
     this.origin = 'https://commerce.coinbase.com';
```

Comment:
out of scope, but this naming is confusing. hostName ought to be origin and origin ought to be... idk commerceRoot or something

Reply:
whatever, bikeshed at this point